lib/Models/CsvCatalogItem.js

```diff
@@ -553,6 +553,7 @@ CsvCatalogItem.prototype.setChartable = function() {
     tableStructure.allowMultiple = true;
     this.isMappable = false;
     tableStructure.getColorCallback = this.getNextColor.bind(this);
+    tableStructure.disableIncompatibleTableColumns = this.disableIncompatibleTableColumns.bind(this);
```

Comment:
Isn't it better to just add `disableIncompatibleTableColumns` here and get only set in `setChartable`, in this way we don't need to pass in an extra options to `DisplayVariablesConcept` ?